src/components/page-header/page-header.component.ts

```diff
@@ -153,4 +156,8 @@ export class PageHeaderComponent {
     select(item: PageHeaderNavigation): void {
         this._pageHeaderService.select(item);
     }
+
+    logoClicked(): void {
+        this.logoClick.emit();
```

Comment:
The `logoClick` type above mentions this should emit a `MouseEvent`, for this to be true, this line here would need to pass the mouse event into the `emit` function. Same with the `backClick`. Currently the type would actually be `EventEmitter<void>` because the emit function is not being passed anything.

Reply:
In fact these two functions can be removed and we can just call them in the html, eg:
```html
<... (click)="logoClick.emit($event)">
```

Reply:
Functions removed and moved with html